cfg/cfg.go

```diff
@@ -53,7 +53,8 @@ var (
 	// UseLocalhostInPluginUrls configures the broker to use the `localhost` name
 	// instead of the Kubernetes service name to build Theia or VSCode plugin
 	// endpoint URL
-	UseLocalhostInPluginUrls bool
+	// True by default, at least as long as all the plugin containers are in the same POD
+	UseLocalhostInPluginUrls = true
```

Comment:
This means that we have no way to disable this mode, but still have a lot of code related to this. Why not add an option to disable/enable it (even though it can be enabled by default) OR remove not needed anymore code?

Reply:
> This means that we have no way to disable this mode

We can disable it when using the broker as a library still

> Why not add an option to disable/enable it 

I could add an option yes, but that didn't seem that necessary and urgent (since there's no use-case for this in Che presently), compared to doing the simplest and smallest change possible.

> remove not needed anymore code

This code would still be useful if, in the future, we decide to add plugins in distinct pods.